### ISSUE TITLE
External DWARF parsing errors

### ISSUE THREAD

**Issue by LostBenjamin** [NationalSecurityAgency/ghidra#4460]
**Describe the bug**
`ghidra` throws an exception when parsing an external DWARF file. It cannot even correctly parse the DWARF version, which is located at the start of each compilation unit in the .debug_info section. The external DWARF file should be well formed, as it can be successfuly parsed by `readelf` or `IDA Pro`.

**To Reproduce**
1. Download and extract [arm.tar.gz](https://github.com/NationalSecurityAgency/ghidra/files/9165732/arm.tar.gz), which consists of the binary `patch` and the external DWARF file in the directory `.build-id`.
2. Loading the binary in `ghidra`.
3. Click `Edit -> DWARF External Debug Config` and set the directory to the one containing `.build-id`.
4. Run the DWARF analyzer by `Analysis -> One Shot -> DWARF`.
5. Wait and see error.

**Expected behavior**
The error message is as follows:
```
Error during DWARFAnalyzer import: Only DWARF version 2, 3, or 4 information is currently supported (detected 31553).
```

**Screenshots**
![image](https://user-images.githubusercontent.com/6451553/180391331-b4d49205-c7c6-4f9e-b851-dd3c2b09d5b9.png)

**Attachments**
I also encountered a similar error on the `x64` version of this binary: [x64.tar.gz](https://github.com/NationalSecurityAgency/ghidra/files/9165820/x64.tar.gz). The error message is as follows:
```
Error during DWARFAnalyzer import: Only DWARF version 2, 3, or 4 information is currently supported (detected 0).
```

**Environment (please complete the following information):**
 - OS: macOS 12.4
 - Java Version: openjdk 11.0.15 2022-04-19
 - Ghidra Version: 10.2
 - Ghidra Origin: locally built from commit `25aa4663d91965416d3e4c96cb9efa71d50d2eac`

**Comment by dev747368** [comment]
initial troubleshooting: it looks like these binaries are using a newer option for compressed Elf sections that we don't support yet.

**Comment by LostBenjamin** [comment]
These binaries are taken from [Ubuntu Debug Symbol Packages](https://wiki.ubuntu.com/Debug%20Symbol%20Packages). `pyelftools` can also parse these binaries. Check [this line](https://github.com/eliben/pyelftools/blob/master/elftools/elf/sections.py#L34) and [this line](https://github.com/eliben/pyelftools/blob/master/elftools/elf/sections.py#L82) from `pyelftools`.

**Comment by c0repwn3r** [comment]
If you are having issues with this, you can use the elfdecompress utility, with the following command: `elfdecompress -vt none [file]` to decompress the sections, which worked for me and I was able to analyze binaries afterward.

**Comment by dev747368** [comment]
Fixed by 0133090b0d84c15c28bf6ea63a14939c07eda71e
